### PR TITLE
fix(build): remove warning

### PR DESCRIPTION
**Pull request by Helias**
Fix this warning
```C++
azerothcore-wotlk/src/common/Configuration/Config.cpp: In member function ‘bool ConfigMgr::LoadAppConfigs(const string&)’:
azerothcore-wotlk/src/common/Configuration/Config.cpp:212:27: warning: zero-length gnu_printf format string [-Wformat-zero-length]
         sLog->outString("");
```

How to test: build the core before and after, the warning will be fixed with this PR.

I don't think that this requires any tests, only the build from travis, so after a quick review I think that we can merge it

### CODE REVIEW

File: src/common/Configuration/Config.cpp [hunk 1]
```diff
@@ -209,7 +209,6 @@ bool ConfigMgr::LoadAppConfigs(std::string const& applicationName /*= "worldserv
     // #2 - Load .conf file
     if (!sConfigMgr->LoadMore(_initConfigFile))
     {
-        sLog->outString("");
```
Comment (FrancescoBorzi) [inline src/common/Configuration/Config.cpp:212]: maybe this was supposed to print a new empty line?

Comment (Kitzunu) [reply src/common/Configuration/Config.cpp:212]: There are a lot of them, esp when loading worldserver, the use I cant understand

Comment (Winfidonarleyan) [reply src/common/Configuration/Config.cpp:212]:
```cpp
sLog->outString();
```

Comment (Helias) [reply src/common/Configuration/Config.cpp:212]: but the reason of this empty sLog? it's just a newline or..?

Comment (Winfidonarleyan) [reply src/common/Configuration/Config.cpp:212]: yes, empty line, cosmetic only

**Comment by Kitzunu** [comment]
Ty for the PR :)

**Pull request merged**